minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -56,7 +56,7 @@ public abstract class AbstractLocalAlluxioCluster {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractLocalAlluxioCluster.class);
 
   private static final Random RANDOM_GENERATOR = new Random();
-  private static final int WAIT_MASTER_START_TIMEOUT_MS = 90_000;
+  private static final int WAIT_MASTER_START_TIMEOUT_MS = 200_000;
```

Comment:
Will this impact on any test cases based on this class?

Reply:
Will check on the results of existing tests to see if there are any impacts

Reply:
Yes, there is failure for alluxio.job.master.JobMasterIntegrationTest.throttleJobWorkerTasks  Time elapsed: 34.072 s  <<< ERROR, maybe need to change to a smaller default timeout value.